plugins/inputs/turbostat/plugin.go

```diff
@@ -0,0 +1,87 @@
+package turbostat
```

Comment:
This plugin is intended only for Linux, so you can safely add `//go:build linux` in every `*.go` file.